api-catalog-ui/frontend/src/components/App/_app.scss

```diff
@@ -111,6 +111,7 @@ body {
         flex-direction: column;
         align-items: center;
         max-width: 368px;
+        min-width: 19.3%;
```

Comment:
Isn't this a duplicate option? (Sonar told me :-) )

Reply:
yes, looks that way, thanks